src/server/schemas/event.schema.js

```diff
@@ -2,19 +2,26 @@ const mongoose = require('mongoose');
 
 const Schema = mongoose.Schema;
 
-const required = type => ({ type, required: true });
```

Comment:
1. If we want to use such short-cuts it's better to create new file for them and use that helper functions in all our Schemas
2. It'll be harder to understand if it'll be mix of function and object declaration

So, I think we should create such helper functions and update all our Schemas to use them.